### PR TITLE
Show me the money  🙌... i mean the data 

### PR DESCRIPTION
**Pull request by karimkawambwa**
## Description

- Adds show data
- Removes the need for them width definition
- Line charts are always vertical 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

## Screenshots

Screen Shot 2020-02-05 at 12 10 52 PM

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: src/components/Card/HURUmapChart.js [hunk 1]
```diff
@@ -75,6 +75,10 @@ function HURUmapChart({
     );
   }
 
+  const rawData = !chartData.isLoading
```
Comment (kilemensi) [inline src/components/Card/HURUmapChart.js]: Any reason `rawData` is not just `data`?

File: src/core/DataTable/exportCSV.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
+function convertToCSV(objArray) {
+  return (typeof objArray !== 'object' ? JSON.parse(objArray) : objArray)
+    .map(innerArray => innerArray.filter(val => val).join(','))
+    .join('\r\n');
+}
+
+export default function exportCSV(headers, items, fileTitle) {
+  if (headers) {
+    items.unshift(headers);
```
Comment (kilemensi) [inline src/core/DataTable/exportCSV.js]: This modifies the original data... Clicking it multiple times leads to repeated header
Screenshot from 2020-02-04 16-56-23

File: src/core/DataTable/exportCSV.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
+function convertToCSV(objArray) {
+  return (typeof objArray !== 'object' ? JSON.parse(objArray) : objArray)
+    .map(innerArray => innerArray.filter(val => val).join(','))
```
Comment (kilemensi) [inline src/core/DataTable/exportCSV.js]: Looks like innerArray is not always an array:
Screenshot from 2020-02-04 17-03-43

**Comment by karimkawambwa** [comment]
@KhadijaMahanga @kilemensi ping

**Pull request merged**